modules/markup/html.go

```diff
@@ -460,28 +460,23 @@ func createEmoji(content, class, name string) *html.Node {
 	return span
 }
 
-func createCustomEmoji(alias, class string) *html.Node {
-
+func createCustomEmoji(alias string) *html.Node {
 	span := &html.Node{
 		Type: html.ElementNode,
 		Data: atom.Span.String(),
 		Attr: []html.Attribute{},
 	}
-	if class != "" {
-		span.Attr = append(span.Attr, html.Attribute{Key: "class", Val: class})
-		span.Attr = append(span.Attr, html.Attribute{Key: "aria-label", Val: alias})
-	}
+	span.Attr = append(span.Attr, html.Attribute{Key: "class", Val: "emoji"})
+	span.Attr = append(span.Attr, html.Attribute{Key: "aria-label", Val: alias})
 
 	img := &html.Node{
 		Type:     html.ElementNode,
 		DataAtom: atom.Img,
 		Data:     "img",
 		Attr:     []html.Attribute{},
 	}
-	if class != "" {
-		img.Attr = append(img.Attr, html.Attribute{Key: "alt", Val: fmt.Sprintf(`:%s:`, alias)})
-		img.Attr = append(img.Attr, html.Attribute{Key: "src", Val: fmt.Sprintf(`%s/assets/img/emoji/%s.png`, setting.StaticURLPrefix, alias)})
-	}
+	img.Attr = append(img.Attr, html.Attribute{Key: "alt", Val: fmt.Sprintf(`:%s:`, alias)})
+	img.Attr = append(img.Attr, html.Attribute{Key: "src", Val: fmt.Sprintf(`%s/assets/img/emoji/%s.png`, setting.StaticURLPrefix, alias)})
```

Comment:
You may or may not want to change this, but it appears that fmt.Sprintf is not very quick or memory efficient and that in go, it generally appears that simple string concatenation is better, even when you need to use `strconv` liberally. Unfortunately it appears that the compiler is not clever enough to just replace `fmt.Sprintf(..., []string)` with the appropriate string concatenations whereas the compiler does appear to avoid the allocations and `copy(...)` for chained `+`.

The main benefit of `fmt.*` functions therefore is if you have `interface{}`, multiple different types or may change the type of some variable in future overloading it with `fmt.Stringer`. (i.e. `type Alias string` and then add a `func (Alias) String() string` method) 

Whilst this is hot code - it's already got plenty of inefficiencies so changing this is unlikely to be a significant improvement but it's something to bear in mind.